lldb/source/Plugins/Process/Windows/Common/NativeThreadWindows.cpp

```diff
@@ -178,9 +178,41 @@ Status NativeThreadWindows::RemoveWatchpoint(lldb::addr_t addr) {
 
 Status NativeThreadWindows::SetHardwareBreakpoint(lldb::addr_t addr,
                                                   size_t size) {
+#if defined(__aarch64__) || defined(_M_ARM64)
```

Comment:
Could we get rid of this ifdef? What would happen if we just let `m_reg_context_up->SetHardwareBreakpoint()` return an error?

Reply:
Yes I think we can. I ll update this.